Uplift/Controllers/HomeViewController+Extensions.swift

```diff
@@ -129,14 +140,25 @@ extension HomeViewController: TodaysClassesListCellDelegate {
 
 }
 
-extension HomeViewController: ProsListCellDelegate {
-
-    func prosListCellShouldOpenProBio(_ proBio: ProBio) {
-        let proDetailViewController = ProBioViewController(pro: proBio)
-        navigationController?.hidesBottomBarWhenPushed = true
-        navigationController?.pushViewController(proDetailViewController, animated: true)
+extension HomeViewController: LookingForListCellDelegate {
+    
+    func lookingForCellShouldTagSearch(at tag: Tag, indexPath: IndexPath) {
+        let cal = Calendar.current
+        let currDate = Date()
+        guard let startDate = cal.date(bySettingHour: 0, minute: 0, second: 0, of: currDate) else { return }
+        let endDate = cal.date(bySettingHour: 23, minute: 59, second: 0, of: currDate) ?? Date()
+        
+        let filterParameters = FilterParameters(endTime: endDate, startTime: startDate, tags: [lookingForCategories[indexPath.row].name])
+        
+        guard let classNavigationController = tabBarController?.viewControllers?[1] as? UINavigationController else { return }
+        guard let classListViewController = classNavigationController.viewControllers[0] as? ClassListViewController else { return }
```

Comment:
you can just chain the 3 guard statements on lines 148, 153, and 154 into one like so:
```
guard let startDate =..., let classNavigationController = ..., let classListViewController = ... else { return }
```